kuvaselaamo/wsgi.py

```diff
@@ -1,6 +1,6 @@
-# -*- coding: utf-8 -*-
```

Comment:
I guess this should be fine to remove here even though it's still a Python 2.7 project.

Reply:
That one was there previously until I replaced the file contents with Helsinki profile's code. I didn't even notice it until now.

If you think that the line is relevant, I can put it back.

Reply:
There's no UTF-8 strings on that file, so you don't have to put it back.